src/frontend/app/shared/components/cards/card-app-uptime/card-app-uptime.component.ts

```diff
@@ -36,5 +37,8 @@ export class CardAppUptimeComponent implements OnInit {
         runningCount: 0
       })
     );
+    this.isRunning$ = this.appService.application$.pipe(
```

Comment:
Do we use this elsewhere  - do we have same code? If so, could move observable to appService